java/app/grapheneos/setupwizard/view/activity/SetupWizardActivity.kt

```diff
@@ -29,8 +29,12 @@ abstract class SetupWizardActivity(
     private var glifLayout: GlifLayout? = null
     private lateinit var activityResultLauncher: ActivityResultLauncher<Intent>
 
-    override fun onCreate(savedInstanceState: Bundle?) {
+    protected fun superOnCreateAtBaseClass(savedInstanceState: Bundle?) {
```

Comment:
any specific use case for this different method?
we can just call `super.onCreate()` in base classes.